### DISCUSSION TITLE
posts/Welcome/

### DISCUSSION THREAD

**Discussion by giscus** [sohzeng/sohzeng.github.io#3] [Announcements]
# posts/Welcome/

This is my first blog! I’m enthusiastically working on building my own website. I’m eager to fill it with captivating content as quickly as I can. I have a strong desire to share my knowledge with others. This website is solely for my personal entertainment, in the hopes of bringing a little brightness to my life.I am currently at a crossroads in my life, hoping to find my way forward. To learn more about me, please click the “ABOUT ME” tab on the left.

https://zengyangyang.com/posts/Welcome/

**Comment by xxyushow** [comment]
Wow
